src/lib/stitching/mergeSchemas.ts

```diff
@@ -64,7 +64,7 @@ export const incrementalMergeSchemas = (testConfig?: any) => {
 
   const vortexSchema = executableVortexSchema()
   schemas.push(vortexSchema)
-  useStitchingEnvironment(vortexStitchingEnvironment())
+  useStitchingEnvironment(vortexStitchingEnvironment(localSchema))
```

Comment:
previously this wasn't needed since we were stitching vortex to metaphysics `artwork` but now under `topArtworks` we are stitching MP `artwork` to vortex.